airbyte-ci/connectors/pipelines/pipelines/dagger/actions/system/docker.py

```diff
@@ -43,7 +63,7 @@ def with_global_dockerd_service(
                     "apk add fuse-overlayfs",
                     # Update daemon config with storage driver.
                     "mkdir /etc/docker",
-                    '(echo {\\"storage-driver\\": \\"fuse-overlayfs\\"} > /etc/docker/daemon.json)',
+                    'echo \'{"storage-driver": "fuse-overlayfs", "registry-mirrors": ["http://172.20.83.84:5000"], "insecure-registries": ["http://172.20.83.84:5000"]}\' > /etc/docker/daemon.json',
```

Comment:
I'm not sure if tailscale is gonna be able to see this service IP, i think it will, but if not then we will front this with the internal load balancer like we do with prefect